<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityDialog</class>
 <widget class="QWidget" name="IdentityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>640</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="identityBarHLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Identity:</string>
       </property>
       <property name="buddy">
        <cstring>m_identityCBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="m_identityCBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="identityButtonsHLayout">
       <item>
        <widget class="QPushButton" name="newBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Duplicate</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_editBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Rename</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_delBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Remove</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalWidget">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="realNameHLayout">
         <item>
          <widget class="QLabel" name="realNameLabel">
           <property name="text">
            <string>&amp;Real name:</string>
           </property>
           <property name="buddy">
            <cstring>m_realNameEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="m_realNameEdit">
           <property name="whatsThis">
            <string>Enter your real name here. IRC is not intended to keep you hidden from your friends or enemies. Keep this in mind if you are tempted to behave maliciously. A fake &quot;real name&quot; can be a good way to mask your gender from all the nerds out there, but the PC you use can always be traced so you will never be truly anonymous.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="nickGroupBox">
         <property name="title">
          <string>Nickname</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="KEditListWidget" name="m_nicknameLBox">
            <property name="checkAtEntering">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="autoIdentifyGroup">
         <property name="title">
          <string>Auto Identify</string>
         </property>
         <layout class="QFormLayout" name="autoIdentifyLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="authTypeLabel">
            <property name="text">
             <string>Type:</string>
            </property>
            <property name="buddy">
             <cstring>m_authTypeCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="m_authTypeCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="nickservNicknameLabel">
            <property name="text">
             <string>Ser&amp;vice:</string>
            </property>
            <property name="buddy">
             <cstring>m_nickservNicknameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="m_nickservNicknameEdit">
            <property name="whatsThis">
             <string>Service name can be &lt;b&gt;&lt;i&gt;nickserv&lt;/i&gt;&lt;/b&gt; or a network-dependent name such as &lt;b&gt;&lt;i&gt;nickserv@services.dal.net&lt;/i&gt;&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="nickservCommandLabel">
            <property name="text">
             <string>Command:</string>
            </property>
            <property name="buddy">
             <cstring>m_nickservCommandEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="m_nickservCommandEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="saslAccountLabel">
            <property name="text">
             <string>Account:</string>
            </property>
            <property name="buddy">
             <cstring>m_saslAccountEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KLineEdit" name="m_saslAccountEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="authPasswordLabel">
            <property name="text">
             <string>Pa&amp;ssword:</string>
            </property>
            <property name="buddy">
             <cstring>m_authPasswordEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="KPasswordLineEdit" name="m_authPasswordEdit">
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="pemClientCertFileLabel">
            <property name="text">
             <string>PEM file:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="KUrlRequester" name="m_pemClientCertFile"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="awayWidget">
      <attribute name="title">
       <string comment="@title:tab">Away</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="m_insertRememberLineOnAwayChBox">
         <property name="whatsThis">
          <string>If you check this box, whenever you perform an &lt;b&gt;/away&lt;/b&gt; command, a horizontal line will appear in the channel, marking the point where you went away. Other IRC users do not see this horizontal line.</string>
         </property>
         <property name="text">
          <string>Mark the last position in chat windows when going away</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="awayMessageLabel">
         <property name="text">
          <string>Default away message:</string>
         </property>
         <property name="buddy">
          <cstring>m_awayMessageEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="m_awayMessageEdit">
         <property name="whatsThis">
          <string>The text you enter here is used as the away message when no message is entered as an argument to the &lt;b&gt;/away&lt;/b&gt; or &lt;b&gt;/aaway&lt;/b&gt; commands, and also when Global Away is toggled from the menu or via the keyboard shortcut.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="awayNicknameLabel">
         <property name="text">
          <string>Away nickname:</string>
         </property>
         <property name="buddy">
          <cstring>m_awayNickEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="m_awayNickEdit">
         <property name="whatsThis">
          <string>Enter a nickname that indicates you are away. Whenever you perform an &lt;b&gt;/away msg&lt;/b&gt; command in any channel joined with this Identity, Konversation will automatically change your nickname to the Away nickname. Other users will be able to tell you are away from your computer. Whenever you perform an &lt;b&gt;/away&lt;/b&gt; command in any channel in which you are away, Konversation will automatically change your nickname back to the original. If you do not wish to automatically change your nickname when going away, leave blank.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="automaticAwayGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>If you check this box, Konversation will automatically set all connections using this Identity away when the screensaver starts or after a period of user inactivity configured below.</string>
         </property>
         <property name="title">
          <string>Automatic Away</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="setAwayAfterLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Set away after</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KPluralHandlingSpinBox" name="m_awayInactivitySpin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="timeOfUserInactivityLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>of user inactivity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="m_automaticUnawayChBox">
            <property name="whatsThis">
             <string>If you check this box, Konversation will automatically cancel away for all connections using this Identity when the screensaver stops or new user activity is detected.</string>
            </property>
            <property name="text">
             <string>Automatically return on activity</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="awayCommandsGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>If you check this box, Konversation will automatically run the away command in all channel tabs using this Identity. &lt;b&gt;%s&lt;/b&gt; in the away command is replaced with the away message. When you return, it will run the return command. &lt;b&gt;%t&lt;/b&gt; in the return command is replaced with the length of time you were away.</string>
         </property>
         <property name="title">
          <string>Away Commands</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="awayLabel">
            <property name="text">
             <string>Away command:</string>
            </property>
            <property name="buddy">
             <cstring>m_awayEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KLineEdit" name="m_awayEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="unAwayLabel">
            <property name="text">
             <string>Return command:</string>
            </property>
            <property name="buddy">
             <cstring>m_unAwayEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="m_unAwayEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="awayWidgetVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>45</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedWidget">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="sCommandLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Pre-shell command:</string>
         </property>
         <property name="buddy">
          <cstring>m_sCommandEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="m_sCommandEdit">
         <property name="whatsThis">
          <string>Here you can enter a command to be executed before connection to server starts&lt;br&gt;If you have multiple servers in this identity this command will be executed for each server</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="codecLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Encoding:</string>
         </property>
         <property name="buddy">
          <cstring>m_codecCBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="m_codecCBox">
         <property name="whatsThis">
          <string>This setting affects how characters you type are encoded for sending to the server. It also affects how messages are displayed. When you first open Konversation, it automatically retrieves this setting from the operating system. If you seem to be having trouble seeing other user's messages correctly, try changing this setting.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="loginLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>I&amp;dent:</string>
         </property>
         <property name="buddy">
          <cstring>m_loginEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="m_loginEdit">
         <property name="whatsThis">
          <string>When you connect, many servers query your computer for an IDENT response. If you computer is not running an IDENT server, this response is sent by Konversation. No spaces are allowed.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="quitLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Quit reason:</string>
         </property>
         <property name="buddy">
          <cstring>m_quitEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KLineEdit" name="m_quitEdit">
         <property name="whatsThis">
          <string>Whenever you leave a server, this message is shown to others.</string>
         </property>
         <property name="text">
          <string>Konversation terminated.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="partLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Part reason:</string>
         </property>
         <property name="buddy">
          <cstring>m_partEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KLineEdit" name="m_partEdit">
         <property name="whatsThis">
          <string>Whenever you leave a channel, this message is sent to the channel.</string>
         </property>
         <property name="text">
          <string>Konversation terminated.</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="kickLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Kick reason:</string>
         </property>
         <property name="buddy">
          <cstring>m_kickEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KLineEdit" name="m_kickEdit">
         <property name="whatsThis">
          <string>Whenever you are kicked from a channel (usually by an IRC operator), this message is sent to the channel.</string>
         </property>
         <property name="text">
          <string>User terminated.</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPasswordLineEdit</class>
   <extends>QWidget</extends>
   <header>kpasswordlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_identityCBox</tabstop>
  <tabstop>newBtn</tabstop>
  <tabstop>copyBtn</tabstop>
  <tabstop>m_editBtn</tabstop>
  <tabstop>m_delBtn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>m_realNameEdit</tabstop>
  <tabstop>m_authTypeCombo</tabstop>
  <tabstop>m_nickservNicknameEdit</tabstop>
  <tabstop>m_nickservCommandEdit</tabstop>
  <tabstop>m_saslAccountEdit</tabstop>
  <tabstop>m_authPasswordEdit</tabstop>
  <tabstop>m_pemClientCertFile</tabstop>
  <tabstop>m_insertRememberLineOnAwayChBox</tabstop>
  <tabstop>m_awayMessageEdit</tabstop>
  <tabstop>m_awayNickEdit</tabstop>
  <tabstop>automaticAwayGroup</tabstop>
  <tabstop>m_awayInactivitySpin</tabstop>
  <tabstop>m_automaticUnawayChBox</tabstop>
  <tabstop>awayCommandsGroup</tabstop>
  <tabstop>m_awayEdit</tabstop>
  <tabstop>m_unAwayEdit</tabstop>
  <tabstop>m_sCommandEdit</tabstop>
  <tabstop>m_codecCBox</tabstop>
  <tabstop>m_loginEdit</tabstop>
  <tabstop>m_quitEdit</tabstop>
  <tabstop>m_partEdit</tabstop>
  <tabstop>m_kickEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
